src/Components/SideBar/SideBar.scss

```diff
@@ -1,170 +1,164 @@
 @import '../../assets/sass/global.scss';
-body {
-  margin: 0;
-  padding: 0;
-  background: #f4f6f9;
-
-  nav {
-    position: fixed;
-    left: 0;
-    top: 0;
-    width: 290px;
+.sidenav {
+  position: fixed;
+  left: 0;
+  top: 0;
+  width: 290px;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  align-self: stretch;
+  height: 740px;
+  background-color: #fff;
+  padding: 100px 10px 0px 10px;
+  .user {
     display: flex;
     flex-direction: column;
-    align-items: flex-start;
-    align-self: stretch;
-    height: 740px;
-    background-color: #fff;
-    padding: 100px 10px 0px 10px;
-    .user {
+    align-self: center;
+    align-items: center;
+    .p-avatar {
       display: flex;
-      flex-direction: column;
-      align-self: center;
+      justify-content: center;
       align-items: center;
-      .p-avatar {
-        display: flex;
-        justify-content: center;
-        align-items: center;
-        width: 90px;
-        height: 90px;
+      width: 90px;
+      height: 90px;
+      border-radius: 50%;
+      border: 2px solid #f46036;
+      .avatar {
+        width: 80px;
+        height: 80px;
         border-radius: 50%;
-        border: 2px solid #f46036;
-        .avatar {
-          width: 80px;
-          height: 80px;
-          border-radius: 50%;
-          box-shadow: 0 2px 5px rgba(0, 0, 0, 0.12);
-          background-size: cover;
-        }
-      }
-      .name {
-        margin-top: 20px;
-        font-weight: 400;
-        font-size: 1em;
-        padding-bottom: 5px;
-        color: #000;
-      }
-      .username {
-        font-weight: 300;
-        font-size: 0.9em;
-        color: #333;
+        box-shadow: 0 2px 5px rgba(0, 0, 0, 0.12);
+        background-size: cover;
       }
     }
-    .info {
-      display: flex;
-      justify-content: space-between;
-      align-items: space-between;
-      align-self: center;
-      width: 100%;
+    .name {
       margin-top: 20px;
-      padding: 0 15px;
-      .item {
-        .label {
-          font-weight: 600;
-          font-size: 0.8em;
-        }
-        .data {
-          text-align: center;
-          font-weight: 300;
-          color: #666;
-          margin-top: 5px;
-          font-size: 0.7em;
-        }
+      font-weight: 400;
+      font-size: 1em;
+      padding-bottom: 5px;
+      color: #000;
```

Comment:
I think you shouldn't hardcode color values, we already have a dark color in the global file

Reply:
👎